### ISSUE TITLE
Editor hangs up

### ISSUE THREAD

**Issue by ankitsny** [froala/wysiwyg-editor#3620]
If i try to paste text which is copied from a large text file then the browser is getting hanged.

tried the same in your website's demo editor and browser hanged.

##### OS.
Ubuntu 18.04

##### Browser.
Chrome 76, 64 bit

Copy paste the content of [Big text file](https://norvig.com/big.txt)

**Comment by casualuser** [comment]
Hello,

We have tried to verify this but in general, it's not possible to copy text from that page - Chrome just hangs continuously and doesn't allow to copy text into clipboard.
So, in this case, it looks like this text sample is too big for Chrome itself.

**Comment by anks333** [comment]
yes, may be the file content is too big for chrome, but we need to figure out how to fix this.

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
